packages/dataviews/src/stories/dataviews.story.tsx

```diff
@@ -615,15 +563,15 @@ export const InfiniteScroll = () => {
 					display: 'block',
 				} }
 			>
-				{ __( 'Infinite Scroll Demo' ) }: { allLoadedRecords.length } of{ ' ' }
-				{ totalItems } items loaded.
+				{ __( 'Infinite Scroll Demo' ) }: { displayData.length } of{ ' ' }
+				{ data.length } items loaded.
 				{ isLoadingMore && __( 'Loading more…' ) }
 				{ ! hasMoreData && __( 'All items loaded!' ) }
```

Comment:
Nitpick and optional, add space between sentence before "All items": `Infinite Scroll Demo: 13 of 37 items loaded.All items loaded!`